src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -148,8 +149,15 @@ public Response getRtFeedDetails(@PathParam("id") int id) {
         List<ViewErrorCountModel> gtfsFeeds;
         Session session = GTFSDB.InitSessionBeginTrans();
         gtfsFeeds = session.createNamedQuery("ErrorCountByrtfeedID", ViewErrorCountModel.class)
-                .setParameter(0, id).setParameter(1, 10).list();
+                .setParameter(0, id).setParameter(1, id).list();
         GTFSDB.commitAndCloseSession(session);
+        for(ViewErrorCountModel viewErrorCountModel: gtfsFeeds) {
+            int index = gtfsFeeds.indexOf(viewErrorCountModel);
+            String formattedTimestamp = getDateFormat(viewErrorCountModel.getLastOccurrence(), id);
+            viewErrorCountModel.setFormattedTimestamp(formattedTimestamp);
+            viewErrorCountModel.setTimeZone(agencyTimezone);
+            viewErrorCountModel.setLastOccurrence(viewErrorCountModel.getLastOccurrence() * 1000); // converting to millis
```

Comment:
I'd leave this as POSIX time, assuming this timestamp is coming directly from the GTFS-rt feed.  We should also add a tooltip when you hover over the column name saying that.